_config.php

```diff
@@ -52,17 +52,19 @@
 // Avoid creating global variables
 call_user_func(function () {
     $module = ModuleLoader::inst()->getManifest()->getModule('silverstripe/admin');
-
-    // Re-enable media dialog
-    TinyMCEConfig::get('cms')
-        ->enablePlugins([
-            'contextmenu' => null,
-            'image' => null,
-            'sslink' => $module->getResource('client/dist/js/TinyMCE_sslink.js'),
-            'sslinkexternal' => $module->getResource('client/dist/js/TinyMCE_sslink-external.js'),
-            'sslinkemail' => $module->getResource('client/dist/js/TinyMCE_sslink-email.js'),
-        ])
-        ->setOption('contextmenu', 'sslink inserttable | cell row column deletetable');
+    
+    if ($module) {
```

Comment:
As @dhensby said we need to determine the underlying cause of this issue rather than simply ignore it when it happens.